renovate.json

```diff
@@ -93,10 +93,6 @@
       "groupName": "MUI Internal",
       "matchPackageNames": ["@mui/internal-*", "@mui/docs"]
     },
-    {
-      "groupName": "Playwright",
-      "matchPackageNames": ["playwright", "@playwright/test", "mcr.microsoft.com/playwright"]
```

Comment:
`playwright` dep no longer exists and `mcr.microsoft.com/playwright` is not used anymore (it was the name of the CircleCI image we used previously for tests)